sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/LogEmitterProviderConfiguration.java

```diff
@@ -0,0 +1,159 @@
+import io.opentelemetry.exporter.otlp.internal.grpc.DefaultGrpcExporterBuilder;
+import io.opentelemetry.exporter.otlp.internal.okhttp.OkHttpExporterBuilder;
+import io.opentelemetry.exporter.otlp.logs.OtlpGrpcLogExporter;
+import io.opentelemetry.exporter.otlp.logs.OtlpGrpcLogExporterBuilder;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigurationException;
+import io.opentelemetry.sdk.logs.SdkLogEmitterProvider;
+import io.opentelemetry.sdk.logs.SdkLogEmitterProviderBuilder;
+import io.opentelemetry.sdk.logs.export.BatchLogProcessor;
+import io.opentelemetry.sdk.logs.export.LogExporter;
+import io.opentelemetry.sdk.resources.Resource;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import javax.annotation.Nullable;
+
+final class LogEmitterProviderConfiguration {
+
+  private static final String EXPORTER_NONE = "none";
+
+  static SdkLogEmitterProvider configureLogEmitterProvider(
+      Resource resource, ConfigProperties config, MeterProvider meterProvider) {
+    SdkLogEmitterProviderBuilder builder = SdkLogEmitterProvider.builder().setResource(resource);
+
+    Map<String, LogExporter> exportersByName = configureLogExporters(config, meterProvider);
+
+    exportersByName.forEach(
+        (name, exporter) -> {
+          BatchLogProcessor batchLogProcessor = BatchLogProcessor.builder(exporter).build();
+          builder.addLogProcessor(batchLogProcessor);
+        });
+
+    SdkLogEmitterProvider logEmitterProvider = builder.build();
+    Runtime.getRuntime().addShutdownHook(new Thread(logEmitterProvider::close));
+    return logEmitterProvider;
+  }
+
+  // Visible for test
+  static Map<String, LogExporter> configureLogExporters(
+      ConfigProperties config, MeterProvider meterProvider) {
+    Set<String> exporterNames = DefaultConfigProperties.getSet(config, "otel.logs.exporter");
+
+    // Default to no exporter
+    if (exporterNames.isEmpty()) {
+      exporterNames = Collections.singleton(EXPORTER_NONE);
+    }
+
+    if (exporterNames.contains(EXPORTER_NONE)) {
+      if (exporterNames.size() > 1) {
+        throw new ConfigurationException(
+            "otel.logs.exporter contains " + EXPORTER_NONE + " along with other exporters");
+      }
+      return Collections.emptyMap();
+    }
+
+    Map<String, LogExporter> exportersByName = new HashMap<>();
+    for (String name : exporterNames) {
+      LogExporter logExporter = configureExporter(name, config, meterProvider);
+      if (logExporter != null) {
+        exportersByName.put(name, logExporter);
+      }
+    }
+
+    return Collections.unmodifiableMap(exportersByName);
+  }
+
+  // Visible for testing
+  @Nullable
+  static LogExporter configureExporter(
+      String name, ConfigProperties config, MeterProvider meterProvider) {
+    switch (name) {
+      case "otlp":
+        return configureOtlpLogs(config, meterProvider);
+      default:
```

Comment:
support for "logging" (SystemOutLogExporter)?

Reply:
Wasn't sure. Its not mentioned as an exporter option in the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#exporter-selection), but none of other the logging exporters we allow are either. 

Can certainly do that if there's no reason not to.

Reply:
I have no problem with it.

Reply:
Included a commit to add support for this.